src/ImageWrapper.cpp

```diff
@@ -9,6 +11,55 @@ extern "C" {
     return reinterpret_cast<Gosu_Image*>( new Gosu::Image(filename, image_flags) );
   };
 
+  Gosu_Image *Gosu_Image_create_from_markup(const char *markup, const char *font, double font_height,
+                                          int width, double spacing, unsigned align, unsigned font_flags, unsigned image_flags)
+  {
+    Gosu::Bitmap bitmap = Gosu::layout_markup(markup, font, font_height, spacing, width,
+                                              (Gosu::Alignment)align, font_flags);
+    return reinterpret_cast<Gosu_Image*>( new Gosu::Image(bitmap, image_flags) );
+  }
+
+  Gosu_Image *Gosu_Image_create_from_text(const char *text, const char *font, double font_height,
+                                          int width, double spacing, unsigned align, unsigned font_flags, unsigned image_flags)
+  {
+    Gosu::Bitmap bitmap = Gosu::layout_text(text, font, font_height, spacing, width,
+                                              (Gosu::Alignment)align, font_flags);
+    return reinterpret_cast<Gosu_Image*>( new Gosu::Image(bitmap, image_flags) );
+  }
+
+  // TODO: replace use of char* with something that doesn't eat NULL
+  Gosu_Image *Gosu_Image_create_from_blob(const char* blob, int columns, int rows, unsigned image_flags)
+  {
+    std::size_t size = columns * rows * 4;
+    Gosu::Bitmap bitmap(columns, rows, Gosu::Color::NONE);
+
+    if (strlen(blob) == size) {
```

Comment:
strlen will not work, since most images will contain null bytes.

Reply:
Oh, I just saw your TODO comment above. Then I'll stop spamming manual TODO comments :D